File: app/src/test/java/com/farmerbb/taskbar/mockito/IntAnswer.java
package com.farmerbb.taskbar.mockito;

import org.mockito.invocation.InvocationOnMock;
import org.mockito.stubbing.Answer;

public class IntAnswer implements Answer<Integer> {
    public int answer = 0;

    @Override
    public Integer answer(InvocationOnMock invocation) {
        return answer;
    }
}

File: app/src/test/java/com/farmerbb/taskbar/mockito/IntAnswer.kt
package com.farmerbb.taskbar.mockito

import org.mockito.invocation.InvocationOnMock
import org.mockito.stubbing.Answer

class IntAnswer : Answer<Int> {
    var answer = 0
    override fun answer(invocation: InvocationOnMock): Int {
        return answer
    }
}
